bindings/java/src/test/java/org/apache/opendal/test/behavior/AbstractBehaviorTest.java

```diff
@@ -605,6 +757,158 @@ public void testBlockingCopyOverwrite() {
         }
     }
 
+    @TestInstance(TestInstance.Lifecycle.PER_CLASS)
+    @Nested
+    class BlockingRenameTest {
+        @BeforeAll
+        public void precondition() {
+            final Capability capability = blockingOperator.info.fullCapability;
+            assumeTrue(
+                    capability.read && capability.write && capability.copy && capability.blocking && capability.rename);
```

Comment:
rename test doesn't depend on copy

Reply:
Ok, does it need to be changed in opendal's behavior test later?

https://github.com/apache/incubator-opendal/blob/3176d21a66052244e7f0a9f05014afd6d075c859/core/tests/behavior/blocking_rename.rs#L25